api/views/feeds.py

```diff
@@ -83,7 +83,7 @@ def item_updateddate(self, item):
         return dateparser.parse(updateddate) if updateddate else None
 
     def item_categories(self, item):
-        categories = [item.get('subject')]
+        categories = item.get('subjects')
```

Comment:
I changed it to follow your suggestion, because may as well, but can many-to-many fields ever return None?